Formula/hatch.rb

```diff
@@ -177,11 +177,8 @@ class Hatch < Formula
   def install
     virtualenv_install_with_resources
 
-    output = Utils.safe_popen_read({ "_HATCH_COMPLETE" => "zsh_source" }, bin/"hatch")
-    (zsh_completion/"_hatch").write output
-
-    output = Utils.safe_popen_read({ "_HATCH_COMPLETE" => "fish_source" }, bin/"hatch")
-    (fish_completion/"hatch.fish").write output
+    (zsh_completion/"_hatch").write Utils.safe_popen_read({ "_HATCH_COMPLETE" => "zsh_source" }, bin/"hatch")
+    (fish_completion/"hatch.fish").write Utils.safe_popen_read({ "_HATCH_COMPLETE" => "fish_source" }, bin/"hatch")
```

Comment:
Let's not change this if we don't have to

Reply:
I changed it to satisfy the to-be-introduced cop:
As we don't support passing custom envs to the new DSL, the cop currently skips `#safe_popen_read` calls with custom envs. However, this only works if `(zsh_completion/"_foo").write` and the `#safe_popen_read` call are on the same line, as its difficult tracking and subsequently skipping something like `(zsh_completion/"_foo").write output`